torch_xla/core/xla_builder.py

```diff
@@ -881,6 +888,7 @@ def abstractify(a):  # make a pytree leaf abstract
         return jax.ShapeDtypeStruct(a.shape, tx.ops.mappings.t2j_dtype(a.dtype))
       return a
 
+    flattened_inputs = list(convert_arg_to_jax(a) for a in flattened_inputs)
```

Comment:
do you know why it worked before when we don't convert the flattened_inputs to jax type?

Reply:
Because dtype is not used as arguments in the existing tests.